File: Python/Programmers/Level 1/없는 숫자 더하기.py
# 0 ~ 9 중에서 numbers에 없는 숫자들의 합을 구하면됨
# def solution(numbers):
#     # 0 ~ 9까지를 set으로 만들고, numbers를 set으로 만든 후 차집합을 구한 이후 sum 함수를 이용하여 합을 구함
#     return sum(set(range(0, 10)) - set(numbers))

# 결국 0 ~ 9까지의 합에서 numbers의 합을 빼면 정답이 됨
def solution(numbers):
    return 45 - sum(numbers)

File: JavaScript/Programmers/Level 1/없는 숫자 더하기.js
// 0 ~ 9 중에 numbers에 없는 숫자를 찾아 더하면 됨
// 0 ~ 9의 합은 45이고 여기서 numbers의 총합을 빼면 없는 숫자들의 합이 됨

function solution(numbers) {
    // 45에서 numbers의 합을 뺌
    return 45 - numbers.reduce((total, number) => total + number, 0);
}
